package-lock.json

```diff
@@ -1555,12 +5039,24 @@
                 "xmlbuilder": "8.2.x"
             }
         },
+        "xtend": {
+            "version": "4.0.1",
+            "resolved": "https://registry.npmjs.org/xtend/-/xtend-4.0.1.tgz",
+            "integrity": "sha1-pcbVMr5lbiPbgg77lDofBJmNY68=",
+            "dev": true
+        },
         "y18n": {
             "version": "4.0.0",
             "resolved": "https://registry.npmjs.org/y18n/-/y18n-4.0.0.tgz",
             "integrity": "sha512-r9S/ZyXu/Xu9q1tYlpsLIsa3EeLXXk0VwlxqTcFRfg9EhMW+17kbt9G0NrgCmhGb5vT2hyhJZLfDGx+7+5Uj/w==",
             "dev": true
         },
+        "yallist": {
+            "version": "3.0.3",
+            "resolved": "https://registry.npmjs.org/yallist/-/yallist-3.0.3.tgz",
+            "integrity": "sha512-S+Zk8DEWE6oKpV+vI3qWkaK+jSbIK86pCwe2IF/xwIpQ8jEuxpw9NyaGjmp9+BoJv5FV2piqCDcoCtStppiq2A==",
+            "dev": true
+        },
         "yargs": {
             "version": "13.2.2",
             "resolved": "https://registry.npmjs.org/yargs/-/yargs-13.2.2.tgz",
```

Comment:
`package-lock.json` should be auto-generated. since there is no change to `dependencies` and `devDependencies` in `package.json`, changes in this file might have been caused by a bad merge. Maybe we could try the following commands in the project's root folder (vscode-ros) and see if there is still any difference:
```batch
rd /s /q node_modules
del package-lock.json
npm install
```

there might still be some difference since we are specifying versions with `^` so it matches for the latest major version, but there should not be this much

Reply:
I think some of the diff in this file is generated when `webpack` and `webpack-cli` were added as dependencies, `package-lock.json` probably did not get updated when those 2 were removed